### ISSUE TITLE
Using Crowdin for translations

### ISSUE THREAD

**Issue by teolemon** [aleene/FoodViewer#770]
Do you want me to setup a separate Crowdin account to enable FoodViewer translations?
Or we can make FoodViewer an official OFF project, and move it to the OFF organisation, and we'll be able to use the OFF account.

**Comment by aleene** [comment]
Yes, I would like to see some extra translations. Maybe it is a good idea to move to OFF. I am not sure what that implies, but I'll see. My only worry is that it will restrict. The app is a bit of my learning/experimenting playground.
